src/test/resources/org/broadinstitute/hellbender/tools/picard/fasta/norm_trunc_fasta.fasta

```diff
@@ -0,0 +1,8 @@
+>chr1
```

Comment:
is this the actual input used? if yes, it should go to a directory picard/fasta/fastq/NormalizeFasta/

if not, remove this file